apps/wallet/src/ui/app/components/receipt-card/index.tsx

```diff
@@ -23,7 +23,7 @@ import { TxnAddress } from '_components/receipt-card/TxnAddress';
 import { TxnAmount } from '_components/receipt-card/TxnAmount';
 import { TxnGasSummery } from '_components/receipt-card/TxnGasSummery';
 import { UnStakeTxnCard } from '_components/receipt-card/UnstakeTxnCard';
-import { getTxnEffectsEventID } from '_components/transactions-card/Transaction';
+import { getTxnEffectsEventID } from '_components/transactions-card';
```

Comment:
design looks off here:

**Figma**
<img width="350" alt="image" src="https://user-images.githubusercontent.com/122397493/216705095-b7308d15-4b92-4c7f-b07f-b5e237482d84.png">

**PR**
<img width="365" alt="image" src="https://user-images.githubusercontent.com/122397493/216705224-f6be438f-087b-4ed3-80a1-bf5b90e30878.png">

Reply:
Thats when the value is 0

Reply:
the top margin looks incorrect when we hide the amount: 

<img width="616" alt="image" src="https://user-images.githubusercontent.com/122397493/216756943-b7dd4b57-8a8c-4904-b508-466594da45c2.png">

Reply:
Done, please review